schema/beneficial-ownership-statements.json

```diff
@@ -75,30 +83,27 @@
           "oneOf": [
             {
               "$ref": "#/definitions/EntityStatementReference"
-            },
-            {
-              "$ref": "#/definitions/EntityStatement"
             }
           ]
         },
         "interestedParty": {
           "title":"Interested party",
           "description":"The interested party has some level of ownership or control over the entity referenced in this beneficial ownership statement.",
+          "type": "object",
           "oneOf": [
             {
-              "$ref": "#/definitions/EntityStatement"
-            },
-            {
-              "$ref": "#/definitions/PersonStatement"
-            },
-            {
-              "$ref": "#/definitions/NullParty"
-            },
-            {
-              "$ref": "#/definitions/EntityStatementReference"
+              "properties": {
+                "entity": {
+                  "$ref": "#/definitions/EntityStatement"
+                }
+              }
             },
             {
-              "$ref": "#/definitions/PersonStatementReference"
+              "properties": {
+                "person": {
+                  "$ref": "#/definitions/PersonStatement"
+                }
+              }
             }
           ]
```

Comment:
We need to either include option of null party here also, or include field at same level as `interestedParty` for declaring why not interested party is known.